product_docs/docs/pem/8/pem_online_help/06a_toc_pem_barman/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: "Monitoring Barman"
+---
+
+Postgres Enterprise Manager (PEM) is designed to assist database administrators, system architects, and performance analysts when administering, monitoring, and tuning PostgreSQL and Advanced Server database servers.
+
+Barman (Backup and Recovery Manager) is an open-source administration tool for remote backups and disaster recovery of PostgreSQL servers in business-critical environments. It relies on PostgreSQL’s Point In Time Recovery technology, allowing DBAs to remotely manage a complete catalogue of backups and the recovery phase of multiple remote servers – all from one location. For more information about Barman, please visit the EDB website at:
+
+  https://www.enterprisedb.com/docs/supported-open-source/barman/
```

Comment:
see previous comment and remember we want to use labels for links instead of showing the url.

Reply:
done in a separate commit